src/auro-datetime.js

```diff
@@ -76,97 +64,84 @@ class AuroDatetime extends LitElement {
     };
   }
 
-  static get styles() {
-    return css`
-      ${styleCss}
-    `;
-  }
-
   /**
-   * @private Internal function generate standard date string
+   * @private Internal function generate date string
    * @returns {string} - date string
    */
   humanDate() {
-    const newDate = new Date();
+    let newDate = new Date();
 
-    return newDate.toLocaleString('en-us', this.dateTemplate);
-  }
+    if (this.utc) {
+      this.dateTemplate.timeZone = 'UTC';
+      newDate = new Date(this.utc);
 
-  /**
-   * @private Internal function generate numeric date string
-   * @returns {string} - date string
-   */
-  numericDate() {
-    this.dateTemplate.month = 'numeric';
-    Reflect.deleteProperty(this.dateTemplate, 'weekday');
-    const newDate = new Date();
+      return newDate.toLocaleString('en-us', this.dateTemplate);
```

Comment:
I think we can remove the return statements in the `if` blocks and just use the one at the end of the function.
```suggestion
```